appsTransport.go

```diff
@@ -43,12 +46,13 @@ func NewAppsTransportKeyFromFile(tr http.RoundTripper, integrationID int, privat
 // installations to ensure reuse of underlying TCP connections.
 //
 // The returned Transport's RoundTrip method is safe to be used concurrently.
-func NewAppsTransport(tr http.RoundTripper, integrationID int, privateKey []byte) (*AppsTransport, error) {
+func NewAppsTransport(ctx context.Context, tr http.RoundTripper, integrationID int, privateKey []byte) (*AppsTransport, error) {
```

Comment:
Contexts are usually request-scoped. Is `NewAppsTransport` really called once to process every requests? From its name, I would imagine it's called once at init time, and then the `AppsTransport`'s `RoundTrip` method is used per request. Is that not how this works?